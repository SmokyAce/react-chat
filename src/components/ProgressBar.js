import React from 'react';
import { LinearProgress } from 'material-ui/Progress';

const ProgressBar = props => (
  <LinearProgress color='secondary' variant='query' {...props} />
);

export default ProgressBar;
