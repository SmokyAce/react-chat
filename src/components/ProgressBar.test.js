/* eslint-env jest */
import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ProgressBar from './ProgressBar';

describe('<ProgressBar />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ProgressBar />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders correctly', () => {
    const tree = renderer.create(<ProgressBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
