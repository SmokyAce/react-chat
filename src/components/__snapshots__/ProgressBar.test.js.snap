// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProgressBar /> renders correctly 1`] = `
<div
  className="MuiLinearProgress-root-18 MuiLinearProgress-colorSecondary-20 MuiLinearProgress-query-22"
  role="progressbar"
>
  <div
    className="MuiLinearProgress-bar-26 MuiLinearProgress-barColorSecondary-28 MuiLinearProgress-bar1Indeterminate-29"
    style={Object {}}
  />
  <div
    className="MuiLinearProgress-bar-26 MuiLinearProgress-barColorSecondary-28 MuiLinearProgress-bar2Indeterminate-32"
    style={Object {}}
  />
</div>
`;
